cli/src/modules/connect.rs

```diff
@@ -9,11 +10,26 @@ impl Connect {
         let ctx = ctx.clone().downcast_arc::<KaspaCli>()?;
         if let Some(wrpc_client) = ctx.wallet().try_wrpc_client().as_ref() {
             let url = argv.first().cloned().or_else(|| ctx.wallet().settings().get(WalletSettings::Server));
-            let network_type = ctx.wallet().network_id()?;
-            let url = url
-                .map(|url| wrpc_client.parse_url_with_network_type(url, network_type.into()).map_err(|e| e.to_string()))
-                .transpose()?;
-            let options = ConnectOptions { block_async_connect: true, strategy: ConnectStrategy::Fallback, url, ..Default::default() };
+            let network_id = ctx.wallet().network_id()?;
+
+            let url = match url.as_deref() {
```

Comment:
That's not an URL anymore

Reply:
Changed here: https://github.com/aspectron/rusty-kaspa/pull/39/commits/f242c8dfa6b5a68e800b66d98e813d686538c725